plugin/src/main/java/io/snyk/eclipse/plugin/preferences/Preferences.java

```diff
@@ -158,11 +159,11 @@ private void migratePreferences() {
 		Set<String> constants = new HashSet<>(Arrays.asList(TRUSTED_FOLDERS, PATH_KEY, ENDPOINT_KEY, INSECURE_KEY,
 				CLI_PATH, CLI_BASE_URL, ACTIVATE_SNYK_CODE_SECURITY, ACTIVATE_SNYK_CODE_QUALITY,
 				ACTIVATE_SNYK_OPEN_SOURCE, ACTIVATE_SNYK_IAC, ADDITIONAL_PARAMETERS, ADDITIONAL_ENVIRONMENT,
-				SEND_ERROR_REPORTS, LSP_VERSION, USE_TOKEN_AUTH, ANALYTICS_PLUGIN_INSTALLED_SENT, ENABLE_DELTA,
-				FILTER_SHOW_CRITICAL, FILTER_SHOW_HIGH, FILTER_SHOW_MEDIUM, FILTER_SHOW_LOW,
-				FILTER_IGNORES_SHOW_OPEN_ISSUES, FILTER_IGNORES_SHOW_IGNORED_ISSUES, FILTER_SHOW_ONLY_FIXABLE,
-				IS_GLOBAL_IGNORES_FEATURE_ENABLED, ENABLE_TELEMETRY, MANAGE_BINARIES_AUTOMATICALLY, ORGANIZATION_KEY,
-				SCANNING_MODE_AUTOMATIC, DEFAULT_ENDPOINT, DEVICE_ID, RELEASE_CHANNEL));
+				SEND_ERROR_REPORTS, LSP_VERSION, ANALYTICS_PLUGIN_INSTALLED_SENT, ENABLE_DELTA, FILTER_SHOW_CRITICAL,
+				FILTER_SHOW_HIGH, FILTER_SHOW_MEDIUM, FILTER_SHOW_LOW, FILTER_IGNORES_SHOW_OPEN_ISSUES,
+				FILTER_IGNORES_SHOW_IGNORED_ISSUES, FILTER_SHOW_ONLY_FIXABLE, IS_GLOBAL_IGNORES_FEATURE_ENABLED,
+				ENABLE_TELEMETRY, MANAGE_BINARIES_AUTOMATICALLY, ORGANIZATION_KEY, SCANNING_MODE_AUTOMATIC,
+				DEFAULT_ENDPOINT, DEVICE_ID, RELEASE_CHANNEL));
```

Comment:
this is used to migrate from secure storage to non-secure storage. changing this may fail migration of an old plugin to the newest version.